src/main/resources/application.conf

```diff
@@ -32,7 +32,7 @@ crypto {
 
 sendgrid {
   apiKey = "dummyApiKey"
-  defaultFromAddress = "help@dev.test.firecloud.org"
```

Comment:
I changed the subdomain. It makes slightly more sense anyway because this address will be used in all non-prod envs